### PR TITLE
Ammar Vehbi - Homework Week 2

### PR DESCRIPTION
**Pull request by ammarvehbi**
Here is a live link to my to-do app: https://rocky-lake-43903.herokuapp.com/

### CODE REVIEW

File: homework-submission/week2/src/App.css [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  text-align: center;
+  width: 275px;
+  margin: 25px;
+  min-height: 150px;
+  padding-top: 5px;
+  padding-bottom: 10px;
+  position: relative;
+  border: 1px solid rgb(255, 215, 7);
+  border-top: 60px solid rgb(255, 215, 7);
+  font-family: 'Reenie Beanie';
```
Comment (nick-barth) [inline homework-submission/week2/src/App.css]: what in the world is reenie beenie

File: homework-submission/week2/src/App.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        todo.done = !todo.done;
+      }
+      return todo;
+    });
+    this.setState({ todos: newTodos });
+  };
+
+  removeTodo = index => {
+    const todos = [...this.state.todos];
+    todos.splice(index, 1);
```
Comment (nick-barth) [inline homework-submission/week2/src/App.js]: nice!

File: homework-submission/week2/src/App.css (hunk 1 above)
Comment (ammarvehbi) [reply homework-submission/week2/src/App.css]: @nick-barth :sweat_smile:  isn't that cool?

File: homework-submission/week2/src/App.js (hunk 2 above)
Comment (ammarvehbi) [reply homework-submission/week2/src/App.js]: :pray:

**Pull request closed without merging**